### PR TITLE
Improve test coverage for bipartite matrix.py

### PR DESCRIPTION
**Pull request by vanshika230**
I have increased test coverage for matrix.py according to here at 95.12% :-
https://app.codecov.io/gh/networkx/networkx/blob/main/networkx%2Falgorithms%2Fbipartite%2Fmatrix.py
The test coverage is now at a 100% as seen below :-
after
I would love a review on this and do any changes if required!

### CODE REVIEW

File: networkx/algorithms/bipartite/tests/test_matrix.py [hunk 1]
```diff
@@ -39,6 +39,11 @@ def test_biadjacency_matrix_order(self):
         M = bipartite.biadjacency_matrix(G, X, Y, weight="weight")
         assert M[1, 2] == 2
 
+    def test_biadjacency_matrix_empty_graph(self):
+        G = nx.empty_graph(2)
+        M = nx.bipartite.biadjacency_matrix(G, [0])
+        assert M == sp.sparse.coo_matrix(([], ([], [])), shape=(1, 1))
```
Comment (rossbar) [inline networkx/algorithms/bipartite/tests/test_matrix.py:45]: It's not particularly clear what sparse matrix will actually result from this call; I'd vote for comparing the result to the expected array directly instead. Also, NX now uses sparse *arrays* not matrices:

```suggestion
        assert np.array_equal(M.toarray(), np.array([[0]]))
```

**Review by rossbar — COMMENTED** [review]
+1 for adding the test, minor suggestion on how to improve it!

File: networkx/algorithms/bipartite/tests/test_matrix.py (hunk 1 above)
Comment (vanshika230) [reply networkx/algorithms/bipartite/tests/test_matrix.py]: I agree, this does make it more clear. Thanks for helping me!!

**Review by rossbar — APPROVED** [review]
LGTM, thanks @vanshika230

**Review by dschult — APPROVED** [review]
This looks good!  Thanks!!

**Pull request merged**